.github/workflows/stale-issue-pr.yml

```diff
@@ -0,0 +1,50 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "30 1 * * *"
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Awaiting response issues
+        uses: actions/stale@v9
+        with:
+          days-before-issue-stale: 14
+          days-before-issue-close: 14
+          stale-issue-label: "stale"
+          # reason for closed the issue default value is not_planned
+          close-issue-reason: completed
+          only-labels: "stat:awaiting response from contributor"
+          stale-issue-message: > 
+            This issue is stale because it has been open for 14 days with no activity.
+            It will be closed if no further activity occurs. Thank you.
+          # List of labels to remove when issues/PRs unstale. 
+          labels-to-remove-when-unstale: "stat:awaiting response from contributor"
+          close-issue-message: >
```

Comment:
Let us only close the issue when contributor has not responded. Let us keep the issue open if the team has not yet responded.

Reply:
For contribution welcome, for stale label to apply it will take 180 days and to close issue it will be 365 days.
Totally, it will be 1.5 years before we close these issues due to any inactivity.
Following the same practice in other keras repo's as well.
Do you still want me to remove this section?